### PR TITLE
syscontainers: by default don't install rpm on AH

### PR DESCRIPTION
**Pull request by giuseppe**
We still don't support it, when --system-package=auto default to no on
Atomic host.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: Atomic/syscontainers.py [hunk 1]
```diff
@@ -167,6 +167,7 @@ def install_user_container(self, image, name):
         return self.install(image, name)
 
     def _install_rpm(self, rpm_file):
```
Comment (yuqi-zhang) [inline Atomic/syscontainers.py]: Is this function actually being used anywhere?

Comment (giuseppe) [inline Atomic/syscontainers.py]: good point, it should be dropped

**Comment by giuseppe** [comment]
@yuqi-zhang a patch to drop it (and a fixup to correct the comment)

**Comment by yuqi-zhang** [comment]
LGTM

**Comment by giuseppe** [comment]
@rh-atomic-bot delegate=yuqi-zhang

**Comment by yuqi-zhang** [comment]
@rh-atomic-bot r+  ba8fbe3

**Pull request closed without merging**